### PR TITLE
Generate ``DateField`` and ``DatTimeField``.

### PR DESCRIPTION
**Pull request by omaciel**
Added code that generates ``DateField`` and ``DatTimeField`` entities.
This will be useful for synchronization plans and schedule tasks in
``Robottelo``.

### CODE REVIEW

**Comment by elyezer** [comment]
Good addition. We also need something that parses the synchronization times like `hourly`

**Comment by omaciel** [comment]
The ``Hourly`` parsing would be handled by the ``entity``.

**Comment by elyezer** [comment]
:+1:

File: nailgun/entity_fields.py [hunk 1]
```diff
@@ -198,10 +200,32 @@ def gen_value(self):
...
 
+    def __init__(self, min_date=None, max_date=None):
+        # If ``None`` is passed then ``FauxFactory`` will deal with it.
+        self.min_date = min_date
+        self.max_date = max_date
+
+    def gen_value(self):
+        """Return a value suitable for a :class:`DateTimeField`."""
+        # pylint:disable=no-self-use
+        # See comment near top of module.
```
Comment (Ichimonji10) [inline nailgun/entity_fields.py]: You should be able to drop these two lines of commenting. This method does make use of `self`.

File: nailgun/entity_fields.py [hunk 2]
```diff
@@ -198,10 +200,32 @@ def gen_value(self):
...
 
+    def __init__(self, min_date=None, max_date=None):
+        # If ``None`` is passed then ``FauxFactory`` will deal with it.
+        self.min_date = min_date
+        self.max_date = max_date
+
+    def gen_value(self):
+        """Return a value suitable for a :class:`DateField`."""
+        # pylint:disable=no-self-use
+        # See comment near top of module.
```
Comment (Ichimonji10) [inline nailgun/entity_fields.py]: You should be able to drop these two lines of commenting. This method does make use of `self`.

**Comment by Ichimonji10** [comment]
ACK pending comments

**Comment by omaciel** [comment]
Comments should be addressed.

**Comment by Ichimonji10** [comment]
Waiting on Travis...

**Pull request merged**